runbotics/runbotics-orchestrator-ui/src/src-landing/views/MainView/index.tsx

```diff
@@ -4,16 +4,19 @@ import Layout from '#src-landing/components/Layout';
 import TempLoginButton from '#src-landing/components/TempLoginButton/TempLoginButton';
 import BenefitsSection from '#src-landing/views/sections/BenefitsSection';
 import HeroSection from '#src-landing/views/sections/HeroSections';
+import IndustriesSection from '#src-landing/views/sections/IndustriesSection';
 import IntegrationSection from '#src-landing/views/sections/IntegrationSections';
 import ProsSection from '#src-landing/views/sections/ProsSection';
 
+
 const MainView = () => (
     <Layout>
         <HeroSection />
         <BenefitsSection />
         <IntegrationSection/>
         <BenefitsSection />
         <ProsSection />
+        <IndustriesSection />
```

Comment:
wrong order i guess. lmove industries under benefits and over Integration section